calculator/src/components/Button.js

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import "./Button.css"
+
+const buttonClass = (value) => {
+  if (value === "AC" || value === "+/-" || value === "%" || value === ".") {
+    return "alternate"
+  } else if (isNaN(value)){
+    return "operator"
+  } else if (value === "0") {
+    return "zero"
+  } else {
+    return null
+  }
+}
+
+const Button = (props) => {
+  return (
+  <div className={`button-wrapper ${buttonClass(props.children)}`} onClick={() => props.handleClick(props.children)}>
```

Comment:
props.children is usually used when when the wrapper div takes in another component as its argument. We also could have made this `div`  a `button` tag. This works for now because the argument `div` which is only innerHTML at this point. Keep that in mind next time.